Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp

```diff
@@ -206,9 +206,10 @@ void SVGTextLayoutEngine::beginTextPathLayout(RenderSVGTextPath& textPath, SVGTe
     float totalLength = lineLayout.m_chunkLayoutBuilder.totalLength();
     unsigned totalCharacters = lineLayout.m_chunkLayoutBuilder.totalCharacters();
 
-    if (textContentElement->lengthAdjust() == SVGLengthAdjustSpacing)
-        m_textPathSpacing = (desiredTextLength - totalLength) / totalCharacters;
-    else
+    if (textContentElement->lengthAdjust() == SVGLengthAdjustSpacing) {
+        if (totalCharacters > 1)
+        m_textPathSpacing = (desiredTextLength - totalLength) / (totalCharacters - 1);
```

Comment:
Wrong indentation here.

Reply:
Will fix it. Waiting on EWS run of if any failing tests. Once all pass, will push new with indentation fix and then will merge. Thanks for review.

Reply:
Fixed.